src/Workspaces/Core/Portable/FindSymbols/ReferenceLocation.cs

```diff
@@ -36,15 +36,21 @@ public struct ReferenceLocation : IComparable<ReferenceLocation>, IEquatable<Ref
         /// </summary>
         public bool IsImplicit { get; }
 
+        /// <summary>
+        /// Inidicates if this is a location where the reference is written to.
+        /// </summary>
+        internal bool IsWrittenTo { get; }
```

Comment:
Note: this is internal because i didn't want to change the public API without approval.  However, i think this would be a generally useful piece of information to expose off of ReferenceLocation.  It's also something we should use when building the FindReferences display.  I.e. reads vs. writes should be called out in the display (possibly using different subnodes).